tests/communication/src/mocks/tls/certhandlermock.hpp

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (C) 2024 Renesas Electronics Corporation.
+ * Copyright (C) 2024 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#ifndef CERTHANDLERMOCK_HPP_
+#define CERTHANDLERMOCK_HPP_
+
+#include <aos/iam/certhandler.hpp>
+
+class MockCertHandler : public aos::iam::certhandler::CertHandlerItf {
```

Comment:
CertHandlerMock

The idea was to reuse mock for different tests. Why you can't use existing `CertHandlerMock`?

Reply:
i did not see that this mock already exist